.gitignore

```diff
@@ -11,6 +11,7 @@ build/windows/installer/certificategenerator/bin
 build/windows/installer/certificategenerator/obj
 # files under amalogswindows dir since this temp directory to build the docker image
 kubernetes/windows/amalogswindows
```

Comment:
Lets not merge into ci_prod until we have full validated. For the image to push, we can have some ev2 script to pull the image from ACR in public cloud and push to AME cloud. lets chat on this tomorrow how to get unblocked on this.